WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginDetailActivity.java

```diff
@@ -117,9 +117,8 @@ public void onCreate(Bundle savedInstanceState) {
         }
 
         mPluginInfo = PluginUtils.getPluginInfo(mPluginStore, mPlugin);
-        if (mPluginInfo == null) {
-            mDispatcher.dispatch(PluginActionBuilder.newFetchPluginInfoAction(mPlugin.getSlug()));
-        }
+        // Always refresh the plugin information to check if there is a newer version
+        mDispatcher.dispatch(PluginActionBuilder.newFetchPluginInfoAction(mPlugin.getSlug()));
```

Comment:
I think we can do this only when `savedInstanceState == null` (ie: when the activity is first created). Otherwise the plugin info is updated every time the device is rotated, which seems wasteful.

Reply:
Fixed in 081ccc6571c18abc787682ba1e2f75fbbb3dbd30.